.github/workflows/publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Pyth SDK to crates.io
+
+on:
+  release:
+    types:
+      - created
+jobs:
+  publish-pyth-sdk:
+    name: Publish Pyth SDK
+    if: ${{ startsWith(github.ref, "pyth-sdk@") }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - run: cargo publish --token ${CARGO_REGISTRY_TOKEN}
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        working-directory: "pyth-sdk"
+  publish-pyth-sdk-solana:
+    name: Publish Pyth SDK Solana
+    if: ${{ startsWith(github.ref, "pyth-sdk-solana@") }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - run: cargo publish --token ${CARGO_REGISTRY_TOKEN}
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        working-directory: "pyth-sdk-solana"
+  publish-pyth-sdk-terra:
```

Comment:
should we rename this to pyth-sdk-cosmos before publishing it?

Reply:
For that we should change dependencies. Considering that terra classic is dead and terra2 is using cosmwasm v1 as well I can make this change. I'll do it in a separate PR and will merge it before this one. So we can also tests it's publishing behaviour.

Reply:
Done now!